src/data.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Address,
+  AddressForm,
+  ContextForm,
+  MetadataSubmission,
+  MultiOmicsForm,
+  StudyForm,
+  SubmissionMetadataCreate,
+} from "./api";
+
+export const defaultAddress = (): Address => ({
```

Comment:
I prefer function names be verbs (generally?). In this case, the verb that comes to mind, while attempting to keep things similarly concise, is `initAddress` (or `initializeAddress` or `resetAddress` or `makeDefaultAddress`). Same comment applies to all other functions in this file.